### PR TITLE
Adding resourceSpecificApplicationPermissions to servicePrincipal in v1.0

### PR DESCRIPTION
**Pull request by psignoret**

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "14162e03-c31b-4eda-92a1-3ac5a125ad25",
+          "ApiChange": "Property",
+          "ChangedApiName": "resourceSpecificApplicationPermissions",
+          "ChangeType": "Addition",
+          "Description": "Added the **resourceSpecificApplicationPermissions** property to [apiApplication](https://docs.microsoft.com/en-us/graph/api/resources/apiApplication?view=graph-rest-1.0) resource.",
+          "Target": "apiApplication"
```
Comment (isvargasmsft) [inline changelog/Microsoft.DirectoryServices.json:11]: This change is missing in this PR. Please add the new property to the apiApplication resource topic.

Comment (psignoret) [reply changelog/Microsoft.DirectoryServices.json]: Oops, thanks. I've removed this entry from the changelog, we're not adding that right now.

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+    {
+      "ChangeList": [
+        {
+          "Id": "14162e03-c31b-4eda-92a1-3ac5a125ad25",
+          "ApiChange": "Resource",
+          "ChangedApiName": "resourceSpecificPermission",
+          "ChangeType": "Addition",
+          "Description": "Added the [resourceSpecificPermission](https://docs.microsoft.com/en-us/graph/api/resources/resourceSpecificPermission?view=graph-rest-1.0) resource type.",
+          "Target": "resourceSpecificPermission"
+        },
```
[the remark this answers is not in the document]
Comment (psignoret) [reply changelog/Microsoft.DirectoryServices.json:12]: Yes, this won't be included at the moment in the application entity. I'll ask for the CSDL to be updated.

File: api-reference/v1.0/resources/serviceprincipal.md [hunk 3]
```diff
@@ -104,6 +104,7 @@ This resource supports using [delta query](/graph/delta-query-overview) to track
 | passwordCredentials | [passwordCredential](passwordcredential.md) collection|The collection of password credentials associated with the application. Not nullable.|
 |preferredSingleSignOnMode|string|Specifies the single sign-on mode configured for this application. Azure AD uses the preferred single sign-on mode to launch the application from Microsoft 365 or the Azure AD My Apps. The supported values are `password`, `saml`, `notSupported`, and `oidc`.|
 |replyUrls|String collection|The URLs that user tokens are sent to for sign in with the associated application, or the redirect URIs that OAuth 2.0 authorization codes and access tokens are sent to for the associated application. Not nullable. |
+|resourceSpecificApplicationPermissions|[resourceSpecificPermission](../resources/resourcespecificpermission.md) collection|The resource-specific application permissions exposed by this application. Currently, resource-specific permissions are only supported for [Teams apps accessing to specific chats and teams](/microsoftteams/platform/graph-api/rsc/resource-specific-consent) using Microsoft Graph. Read-only.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/serviceprincipal.md:107]: Could you please also update the JSON representation on this file to include this property. 

In addition, please include this property in the example request/responses where this entity is returned (if applicable), for example the GET and UPDATE operations.

Comment (psignoret) [reply api-reference/v1.0/resources/serviceprincipal.md:107]: I've added the JSON representation and included this in serviceprincipal-get.md.

**Review by isvargasmsft — APPROVED** [review]

**Comment by Lauragra** [comment]
@psignoret , is this PR ready to merge? The schema associated with this PR has been release so we need to get the docs published. Please set the **ready to merge** label when ready. Thanks.

**Comment by psignoret** [comment]
@lauragra Indeed, it was just added and I was waiting for confirmation. Thanks!

**Pull request merged**